sycl/doc/extensions/experimental/sycl_ext_intel_runtime_buffer_location.asciidoc

```diff
@@ -0,0 +1,119 @@
+= SYCL_INTEL_usm_runtime_properties
```

Comment:
We are standardizing on a new template for extensions.  Can you use the template in #5663 for this extension?  In order to do that, you will need to decide:

* Is this a "oneapi" or an "intel" extension?
* Is this a "supported" or an "experimental" extension?

Reply:
Thanks for the pointer to template!

Currently this extension will be in intel extension (let me know if it makes more sense to be an oneapi extension

It is also an experimental extension given this is a workaround.